src/devices/bus/qbus/qbus.h

```diff
@@ -38,6 +38,8 @@ class device_qbus_card_interface : public device_interface
 	// construction/destruction
 	device_qbus_card_interface(const machine_config &mconfig, device_t &device);
 
+	virtual void device_reset() { }
+
```

Comment:
You’re supposed to override `interface_pre_reset` or `interface_post_reset` to add reset hooks for mixins.  This is going to cause problems with ambiguous member function names as devices will inherit an identically named member function from `device_t` and `device_qbus_card_interface`.